improved documentation

pinak22dhir opened the pull request:
## Pull Request

added documentation

AnupK1234 commented:
Kindly use the [Pull Request template](https://github.com/AnupK1234/MealMatters/blob/main/.github/PULL_REQUEST_TEMPLATE.md) while making a PR.

AnupK1234 commented:
> Kindly use the [Pull Request template](https://github.com/AnupK1234/MealMatters/blob/main/.github/PULL_REQUEST_TEMPLATE.md) while making a PR.

@pinak22dhir Are you done with the PR ?

AnupK1234 commented:
> @AnupK1234 what do mean by pr template ? How can I access it?

@pinak22dhir @pinak-dhir 
You can find the PR Template here : [Link](https://github.com/AnupK1234/MealMatters/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
Fill all the applicable details along with **issue no**. assigned to you

pinak22dhir commented:
@AnupK1234 as pr of this is done, should i do another pr in another branch as it is not prefering me that template as i did pr.

pinak22dhir commented:
Pull Request
Related Issues: [List any related issues or reference them using the syntax #issue_number.]

Fixes #2

Description: [changes the documentation.]

Checklist:

 I have tested my changes.
 My code follows the project's coding standards.
 I have updated the documentation ,added code of conduct.md 


Additional information
code_of_conduct.md is the guidelines for contributors to follow, in case of any violation, strict action will be taken against
